src/widgets/histogram/histogram-title-view.js

```diff
@@ -50,12 +54,59 @@ module.exports = cdb.core.View.extend({
     this.addView(sizesTooltip);
   },
 
-  _applySizes: function () {
-    this.dataviewModel.set('histogram_sizes', true);
+  autoStyle: function () {
+    var data = this.dataviewModel.get('data');
```

Comment:
this method should be outside of here, it's likely we will need this in other parts.

It should be something like

```
var cartocss = pecanByHistogram(histogram, stylingOtions [, geometryType])
```